synedrion/src/tools.rs

```diff
@@ -5,7 +5,7 @@ pub(crate) mod hashing;
 mod hide_debug;
 pub(crate) mod sss;
 
-pub(crate) use hide_debug::HideDebug;
+pub(crate) use hide_debug::{HideDebug, Secret};
```

Comment:
The module name should probably change now that `Secret` is included.

Reply:
It'll be renamed in #163. I moved as much of the `Secret` stuff there, but some still had to be kept because it's just too much work to extract it from the PR

Reply:
Moved to `tools::secret`